recipes/beyla-service-graph/collector-config.yaml

```diff
@@ -53,47 +48,63 @@ spec:
         detectors: [env, gcp]
         timeout: 2s
         override: false
-      # Move server.address from a metric attribute to a resource attribute so we can use it to get k8s attributes.
+      # Move server and client address from metric attributes to resource attributes so we can
+      # use it to get k8s.pod.name for the server and client pods
       groupbyattrs:
         keys:
           - server.address
-      # Assume server.address is a pod IP address
-      resource:
-        attributes:
-          - key: k8s.pod.ip
-            from_attribute: server.address
-            action: insert
-      k8sattributes:
+          - client.address
+      transform/serveraddress_to_k8spodip:
+        metric_statements:
+          - context: resource
+            statements:
+            - delete_key(attributes, "k8s.pod.name")
```

Comment:
are we just re-doing work already done by beyla?  Why do we need to delete this attribute?

Reply:
I was hoping to remove the annotation being done by Beyla altogether once that bug is fixed. I could re-use this saved attribute in theory and remove the `k8sattributes/1`. Wdyt?

Reply:
up to you.  A comment explaining what the plan is would be helpful for future reference